tests/cohort_tests.py

```diff
@@ -0,0 +1,40 @@
+"""
+See
+https://github.com/NHSDigital/pytest-nhsd-apim/blob/main/tests/test_examples.py
+for more ideas on how to test the authorization of your API.
+"""
+import requests
+import pytest
+import json
+from lib import Generators
+from lib.constants import CORRELATION_IDS
+from lib.api_helpers import *
+
+
+@pytest.mark.functionaltest
+def test_for_getall_query(nhsd_apim_proxy_url):
+    published_cohort_definitions_response = requests.post(
+        f"{nhsd_apim_proxy_url}/api",
+        headers=Generators.generate_target_server_headers("76491414-d0cf-4655-ae20-a4d1368472f3"),
+        json=published_cohort_library_get_all_request_body
+    )
+
+    published_cohort_definitions_Response_json = json.dumps(published_cohort_definitions_response.json(), indent=1)
+    expected_response_published_library_get_all = json.dumps(
+        json.load(published_library_get_all_json_file_location), indent=1)
```

Comment:
You don't need to use dumps() as your data is already in a json format.

You simply do:
```
published_cohort_definitions_response = published_cohort_definitions_response.json()
    expected_response_published_library_get_all = json.load(published_library_get_all_json_file_location)
```

and then assert those 2
```
assert expected_response_published_library_get_all == published_cohort_definitions_response_json
```

Reply:
the json response from published_cohort_definitions_response.json() appears as below used indent 1 to make it formatted better
![image](https://github.com/NHSDigital/caas-published-cohort-definitions/assets/120654585/e00e9eda-5e85-491f-b88c-d09d65b5b07e)

Reply:
We can create a separate beautifier function that could be useful when debugging, but in our case, there isn't anything visual, and I'd stick with fewer lines of code as the result is exactly the same.